src/arrayMethodSort.js

```diff
@@ -4,8 +4,27 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    let callBack = compareFunction;
+
+    if (!callBack) {
+      callBack = function (a, b) {
+        return String(a) > String(b) ? 1 : -1;
+      };
+    }
+
+    for (let i = 0; i < this.length - 1; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (callBack(this[i], this[j]) > 0) {
+          const backup = this[i];
+
+          this[i] = this[j];
+          this[j] = backup;
```

Comment:
Nice job on correctly swapping the elements in the array. However, you could also use array destructuring to swap the elements without needing a temporary variable. Here is how you can do it:
```javascript
if (callBack(this[i], this[j]) > 0) {
  [this[i], this[j]] = [this[j], this[i]];
}
```
This is just a suggestion and not a requirement. Your current code works perfectly fine.